apps/wallet-connect/src/assets/qr-code.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
https://components.gnosis-safe.io/?path=/docs/data-display-icon--icons

There is already `qrCode` here

Reply:
yeah, that's the one currently in prod but the @liliiaorlenko one is different, for this reason I imported it here as SVG

Reply:
Should we use the library one anyway?